internal/store/bundle.go

```diff
@@ -202,6 +329,30 @@ func (b *bundleStore) pathToReference(path string) (reference.Named, error) {
 	return reference.ParseNamed(name)
 }
 
+func appendRef(refsMap referencesMap, id ID, ref reference.Reference) {
```

Comment:
`appendRef` and `removeRef` could be methods added to the `referencesMap` type.

Reply:
Nice catch. I replaced the raw map by this type at the very end and didn't think to add the functions